spec/features/course/assessment/submission/worksheet_spec.rb

```diff
@@ -78,6 +78,28 @@
         end
       end
 
+      scenario 'I can edit my existing comment', js: true do
```

Comment:
It's a tradeoff. We want the test to be fast so that it will boost our development. But large test case are hard to read and refactor, thus it reduces the maintainability. 

I am good to merge them in feature test especially for javascript tests, but we should add some comments to each section so that we will be clear what the test is about. And for unit test, let's still do a single concept per test case.